src/components/LayoutPanels/library/dom/dragToResizePanelWidth.ts

```diff
@@ -23,70 +25,85 @@ export function dragToResizePanelWidth({
     throw new Error("divRef.current is not defined");
   }
 
```

Comment:
A lot of this code is just moved around and renamed. Ill point out the meat of it below